src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java

```diff
@@ -37,6 +39,7 @@ public class ResourceModificationEventListener {
   @TransactionalEventListener
   public void afterCreate(ResourceCreatedEvent resourceCreatedEvent) {
     log.info("ResourceCreatedEvent received [{}]", resourceCreatedEvent);
+    entityManager.detach(resourceRepository.getReferenceById(resourceCreatedEvent.id()));
```

Comment:
Could you please clarify why detach is needed before reading the same resource from repository? 
@TranscationalEventListener has TransactionPhase.AFTER_COMMIT as default value which means that initial transaction was committed.

Reply:
the changes are stale
after yesterday's discussion decided to refactor the code